.github/workflows/publish-next.yml

```diff
@@ -0,0 +1,35 @@
+name: publish-next
+
+on:
+  release:
+    types: [prereleased]
+
+jobs:
+  publish-npm-registry:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          cache: 'yarn'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Remove example code
+        run: rm -rf IapExample
+
+      - name: Install dependencies
+        run: yarn install --immutable
+
+      - name: Run lint scripts
+        run: yarn lint:ci
+
+      - name: Verify no files have changed after auto-fix
+        run: git diff -- ":(exclude)IapExample/*" --exit-code HEAD
+
+      - run: npm publish --tag next
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
`publish-next.yml` file should not be merged to `main`.

When publishing `main` package, the `next` package will be published either.
I am not currently sure of the better way to separate the publication between `main` and `next`.

My current solution is to put different publication files in each branch so they must not live together.

Reply:
I'm going to use it on this branch to create a new RC

Reply:
@hyochan  Please take a look at https://github.com/dooboolab/react-native-iap/discussions/1852

Reply:
I have removed the file so this can become 9.0.0